packages/cloudflare/src/index.ts

```diff
@@ -47,7 +52,7 @@ type Options = {
 	 * 'local': use a static req.cf object, and env vars defined in wrangler.toml & .dev.vars (astro dev is enough)
 	 * 'remote': use a dynamic real-live req.cf object, and env vars defined in wrangler.toml & .dev.vars (astro dev is enough)
 	 */
```

Comment:
The comment could mention that we are currently restructuring and want to support only the option object in the future.